content/en/snippets/others/xhr_get_post.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Send GET / POST request synchronously"
+weight: 10
+ie_support: true
+---
+Use these snippets when you want to send a HTTP request to an API endpoint.
+
+The values set in the header and body are examples. The actual values depend on the specification of the API being requested. Please set appropriate values.
+
+<a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest" target="_blank">XMLHttpRequest</a> is used to send HTTP requests.
+
+## Send GET request to a URL
+
+```js
+var url = "API URL";
+var xhr = new XMLHttpRequest();
+
+xhr.open("GET", url, false);
+xhr.setRequestHeader("Accept", "application/json");
```

Comment:
Quick question here ✋ Why is the header required only when it's a GET request?

Reply:
@tsuemura 
Thanks for your review.
This is a just sample of setting header.
I deleted this line and updated description because the header depends on the API specification.
[32ef6d1](https://github.com/autifyhq/autify-javascript-snippets/pull/47/commits/32ef6d1eae12c632a434667ca47a9ef154db5664)